src/app/leaves/components/LeaveActionRender.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useMemo } from "react";
+import ActionsMenu from "@/common/ActionMenu/ActionsMenu";
+import {
+  Info as InfoIcon,
+  Trash2 as TrashIcon,
+  Edit as EditIcon,
+  CheckCircle as CheckCircleIcon,
+  AlertCircle as AlertCircleIcon,
+} from "lucide-react";
+import { MemberRole } from "@/lib/constants";
+
+const loggedUser = {
+  name: "John Smith",
+  role: "member",
+};
+
+const LeaveTableActionRender = ({
+  leaveRequestStatus,
+}: {
+  leaveRequestStatus: string | undefined;
+}) => {
+  const handleViewDetails = () => {
```

Comment:
Do we have the ticket to handle all these missing implementations?

Reply:
We still haven't created any tickets regarding the implementation of these handles.